exporter/exporterhelper/internal/retry_sender.go

```diff
@@ -65,7 +65,7 @@ func (rs *retrySender) Shutdown(context.Context) error {
 	return nil
 }
 
-// send implements the requestSender interface
+// Send implements the requestSender interface
 func (rs *retrySender) Send(ctx context.Context, req internal.Request) error {
 	// Do not use NewExponentialBackOff since it calls Reset and the code here must
```

Comment:
Since backoff v5 does not call `Reset` anymore, this comment should be removed. We could also theoretically now use the `NewExponentialBackoff` function (not sure if it brings us any value).

Reply:
Done.